.nycrc-frontend.json

```diff
@@ -1,9 +1,10 @@
 {
   "reporter": ["lcov"],
-  "all": true,
+  "all": false,
```

Comment:
What's the thinking behind disabling this option? If I recall correctly this setting controls whether to count source files even if they aren't required by tests (which I figure would give us a more accurate coverage reading).

Reply:
Right, so it doesn't work with `instrument: false`. See https://github.com/badges/shields/pull/2906/files#diff-a232581bf91df05dba97d6a2b92ebb3d which patches around the situation and fixes #2616.

The limitation is poorly documented and I started to track down the loose ends and contribute documentation upstream, but I don't understand it quite well enough to articulate it clearly.

Reply:
I thought I remembered an issue with frontend coverage. That articulation is good enough to cover the why 👍